skimage/feature/register_translation.py

```diff
@@ -58,7 +58,49 @@ def _upsampled_dft(data, upsampled_region_size,
             raise ValueError("number of axis offsets must be equal to input "
                              "data's number of dimensions.")
 
-    im2pi = 1j * 2 * np.pi
+    # For the 2D case, it is more efficient to use the well-optimized
```

Comment:
Is this intended change to be here?

Reply:
ping @hmaarrfk something weird in your history...

Reply:
Yeah. I'm really not sure. I didn't have time to figure out why the rebase went astray

Reply:
Thahnks you two.  I think what happened was that that function was simplified. I was more careful with my rebase now and I think all is well.